packages/ai-ide/src/browser/style/index.css

```diff
@@ -348,3 +348,104 @@
   font-weight: bolder;
   font-style: normal;
 }
+
+/* Token Usage Configuration Styles */
+.token-usage-configuration-container {
+  padding: 16px;
+}
+
+.token-usage-configuration-title {
+  margin: 0 0 16px 0;
+  border-bottom: 1px solid var(--theia-panelTitle-activeBorder);
+  padding-bottom: 8px;
+}
+
+.token-usage-controls {
+  display: flex;
+  justify-content: flex-end;
+  margin-bottom: 16px;
+}
+
+.token-usage-refresh-button {
+  display: inline-flex;
+  align-items: center;
+  background-color: var(--theia-button-background);
+  color: var(--theia-button-foreground);
+  border: none;
+  border-radius: 4px;
+  padding: 6px 12px;
+  font-size: 12px;
+  cursor: pointer;
+  transition: background-color 0.2s;
+}
+
+.token-usage-refresh-button:hover {
+  background-color: var(--theia-button-hoverBackground);
+}
+
+.token-usage-refresh-button i {
+  margin-right: 6px;
+}
+
+.token-usage-table-container {
+  overflow-x: auto;
+  border: 1px solid var(--theia-panelTitle-activeBorder);
+  border-radius: 6px;
+  background-color: var(--theia-editorWidget-background);
+}
+
+.token-usage-table {
+  width: 100%;
+  border-collapse: collapse;
+}
+
+.token-usage-header th {
+  text-align: left;
+  padding: 10px;
+  background-color: var(--theia-editor-background);
+  border-bottom: 1px solid var(--theia-panelTitle-activeBorder);
```

Comment:
I think we shouldn't use the active border here. This color is somewhat reserved for components that have focus and are active. Here we use this color all the time.